### DISCUSSION TITLE
This post is regarding JSON schemas

### DISCUSSION THREAD

**Discussion by Bhuvi-Naveen** [c-jimenez/open-ocpp#126] [General]
Hi,

Thanks to provide information on below JSON Schemas. 

I would like to know, what was the source for generating this schemas, as I was not able to find in official release of OCPP 1.6J. It would be great if could refer to the source.
 
CertificateSigned.json
DeleteCertificate.json
ExtendedTriggerMessage.json
GetInstalledCertificatelds.json
GetLog.json
InstallCertificate.json
LogStatusNotification.json
SecurityEventNotification.json
SignCertificate.json
SignedFirmwareStatusNotification.json
SignedUpdateFirmware.json

**Comment by c-jimenez** [comment]
Hi,

I also did not found any official inputs from OCA for these messages.

I generated these files manually from the corresponding OCPP2.0.1 JSON specifications from OCA which I have adapted to match the OCPP1.6 white paper  specifications.

**Comment by Bhuvi-Naveen** [comment]
Hi c-jimenez,

Thank you
